### ISSUE TITLE
Add option for concurrent downloads of songs from a playlist

### ISSUE THREAD

**Issue by abawer** [ytdl-org/youtube-dl#31511]
Would be nice to have option do download songs concurrently from a playlist.
Currently the default behaviour for downloading a playlist is to serial download of song by song.

**Comment by dirkf** [comment]
This isn't and probably won't be supported directly.

Generally the limiting factor in downloading is the internet connection, although there is scope for overlapping download of item n+1 with post-processing of item n. So downloading several items in parallel may not have much effect, except to invite blocking if they're from the same site.

Using a script, you could list the URLs of the playlist items and download them using separate yt-dl instances. For example, write the URL list to a file, split that file into N chunks, invoke yt-dl instances with `--batch-file chunk1` ... `--batch-file chunkN`.

See #3746.
